client/src/router.jsx

```diff
@@ -0,0 +1,23 @@
+import {
+  createBrowserRouter,
+} from 'react-router-dom';
+
+import {Error} from './pages';
+import { Layout } from './pages/Layout';
+
+
+const router = createBrowserRouter([
+  {
+    path: '/',
+    element: <Layout/>,
+    errorElement: <Error />,
+    children: [
+      {
+        index: true,
+        element: <><p>Hello World</p></>
```

Comment:
Add app.tsx in this route

Reply:
Disagree with this solution. It is better to add RouterProvider into App, to have opportunity to extend App component in future